R/variable-description.R

```diff
@@ -0,0 +1,15 @@
+#' Variables from registers and their descriptions that are required for the
+#' OSDC algorithm.
```

Comment:
OSDC is capitalised here. Should we do this everywhere?

Reply:
It is an abbreviation, so I suppose we should when refering to the algorithm, but keep the package name as lower-case. That's how I did it in the validation paper, but I have no strong feelings about it.

Reply:
I'll resolve this based on #61